concordia/templates/registration/password_reset_complete.html

```diff
@@ -8,11 +8,7 @@
 <div class="container p-3">
     <div class="row">
         <div class="col-md-6 mx-auto">
-            <p>{% trans "Your password has been set.  You may go ahead and log in now." %}</p>
-
-            <p class="text-center">
-                <a class="btn btn-primary" href="{% url 'login' %}">{% trans 'Log in' %}</a>
-            </p>
+            <p>{% trans "Your password has been reset and you have been logged in. If your account was inactive, it has been activated." %}</p>
```

Comment:
@elainekamlley @lalgee Opportunity for better wording here and maybe also welcome guide link.

Reply:
Your password has been reset and you are now logged in. If your account was inactive, it has been activated.

New here? Visit the By the People Welcome Guide link) for instructions and help getting started. Or ,jump right in (link to 1st page in need of transcription) to a page that needs your help!